src/devices/Common/IIntegratedLed.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Drawing;
+
+namespace Iot.Device
+{
+    public abstract class IntegratedLed : IDisposable
+    {
+        /// <summary>
+        /// Color correction parameter
+        /// </summary>
+        public double Gamma { get; set; } = 2.2;
+
+        /// <summary>
+        /// Colors of LEDs
+        /// </summary>
+        public Span<Color> Pixels => _pixels;
+
+        public abstract void Flush();
+
+        protected Color[] _pixels;
```

Comment:
Protected members should follow the same naming conventions as public: `Pixels`, Honestly I'd recommend to just make this public rather than artificially wrapping the other member with Span for no real benefit (use Span when it can save allocations, if it's already allocated then there is no really point in doing so unless you plan to hide some implementation detail perhaps)